blog/2020-01-everything-github/README.md

```diff
@@ -0,0 +1,706 @@
+But the the usage of a private repos with Github Actions and Github Pages is only possible from the pro plan and higher.
+But the big advantage is that we do not have to set up a token in a tiresome way.
+The `GITHUB_TOKEN` token is already there and we do not have to do at all.
+
+Behind the scenes, Github reuses an existing feature, that is known as GitHub apps.
+In contrast to programms that authenticate with a "personal access token",
+apps do not run on behalf of a specific user - which is great.
+To make this happen, GitHub automatically installs a GitHub App on the repository and makes a corresponding "installation access token" with the name `GITHUB_TOKEN` available as a secret. 
+GitHub fetches a new token for each job, before the job begins.
+Awesome!
+
+> **ℹ️ Note**
+> 
+> As already mentioned above, the `GITHUB_TOKEN` (installation access token) will only trigger a release of a new website if the action runs in a **private repository**. In a public repo, a commit is generated, but the site does not change. See this [GitHub Community post](https://github.community/t5/GitHub-Actions/Github-action-not-triggering-gh-pages-upon-push/m-p/26869) for more info. If your repo is public, you must still use the `GH_TOKEN` (personal access token).
+
+If you have not already done so, please read the text on general workflow usage.
+The workflow looks very similar, but now we must use the other token.
+Here is a shorter example, without the tests:
+following example:
+
+```yml
+name: Deploy to GitHub Pages via angular-cli-ghpages
+
+on: [push]
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Use Node.js 10.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: 10.x
+
+    - name: Prepare and deploy
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        npm install
+        npm run ng -- deploy --base-href=/everything-github-demo/ --name="<Your Git Username>" --email=your.mail@example.org
+```
+
+**Et voila!!** 🚀
+
+This is all it takes to deploy the latest version of the Angular App to GitHub Pages with each push.
+
+
+## 6. Extra: Custom Domains
+
+Finally, we would like to shortly describe how you use your own domain.
+GitHub Pages supports custom domains, by placing a specific file in the root directory of the `gh-pages` branch – and `angular-cli-ghpages` will create that file for you.
+<!-- GitHub also provides [detailed instructions for this](https://help.github.com/en/github/working-with-github-pages/about-custom-domains-and-github-pages). -->
+
+### 6.1 Configuring a subdomain
+
+Setting up a subdomain isn't very hard and if you already own a main domain, then you don't need to make a new investment.
+In our example our app should be accessible under the domain `everything-github-demo.angular.schule`.
+
+1. First we have to change the DNS for the domain by setting up a so-called CNAME record.
```

Comment:
CNAME record in italic? As it will be used the first time with "so called"

Reply:
i don't like italic! ^^

Reply:
so-called `CNAME` record